src/extensions/WorkspaceView/SortingView.tsx

```diff
@@ -131,7 +140,7 @@ const SortingView: React.FunctionComponent<Props> = (props) => {
           {`Sorting: `}
           {sorting.sortingLabel}
           {` | Recording: `}
-          {recording.recordingLabel}
+          {<Hyperlink onClick={handleGotoRecording}>{recording.recordingLabel}</Hyperlink>}
```

Comment:
I had honestly never noticed the sorting label before where it is. Might want to consider moving it to the top of the page (below the purple header bar) so that there's a bit more location consistency between this and the similar navigation "Back to Recordings" link on the recording page. Maybe not, space is at a premium; it'd just make the navigation experience more straightforward.

Might be informative to ask the FrankLab group what they think, too.

Reply:
I agree, it's more intuitive at the top. On the other hand, it doesn't look as nice, IMO. So we'll postpone this decision until we can get an expert design UX makeover.